src/assets/javascripts/angular/configuration/routeConfiguration.js

```diff
@@ -125,6 +125,20 @@ angular.module('calcentral.config').config(function($routeProvider) {
     pageName: 'Student Overview'
   });
 
+  const reactPaths = [
+    '/service_alerts',
+    '/service_alerts/new',
+    '/service_alerts/:id/edit',
+    '/user_auths',
+    '/user_auths/new',
+    '/user_auths/:id/edit',
+  ]
+
+  reactPaths.forEach(path => $routeProvider.when(path, {
+    templateUrl: 'react.html',
+    controller: 'ReactController'
```

Comment:
@developish - Why are we calling this 'ReactController'? Is it because you decided to create this under your pure "React" environment space that we can use to demonstrate if a component works without Angular dependencies?

Reply:
Brandon clarified that under our AngularJS environment that we rely on in PROD, this is used to provide a purer React environment for new pages that we add.